src/Formula/Parser.elm

```diff
@@ -141,6 +137,24 @@ quantified symbols constructor =
         |= lazy (\_ -> formula)
 
 
+eq : Parser Formula
+eq = succeed EqAtom
+    |= term
+    |. spaces
+    |. oneOfSymbols [ "≐", "=" ]
+    |. spaces
+    |= term
+
+
+negEq : Parser Formula
+negEq = succeed EqAtom
```

Comment:
It's confusing to split the processing of non-equality like this. You can write `succeed (Neg << EqAtom)` here and replace `succeed Neg |= negEq` with just `succeed <| negEq` in `formula`.